core/agents/interfaces.go

```diff
@@ -13,6 +13,21 @@ type Interface interface {
 	AgentName() string
 }
 
+type AlbumInfo struct {
+	Name         string
+	MBID         string
+	Description  string
+	URL          string
+	SmallImgUrl  string
+	MediumImgUrl string
+	LargeImgUrl  string
```

Comment:
Maybe? I returned three because that's what the subsonic API requires (and I didn't want to add more columns to the database). If we return all the sizes, then it could also be difficult to disambiguate from different sources. For example, Last.fm returns five sizes, all stringly typed, while the Cover Art Archive returns integer (250, 500, 1200) and string (aliased to the integer sizes)

Reply:
Remember that Navidrome features are not exclusively tied to Subsonic API, we can (and will) expose these functionality using other APIs, including the Native API. See also my comment above.